client/src/main/java/jake2/client/CL.java

```diff
@@ -344,18 +347,13 @@ static void SendKeyEvents() {
             return;
         }
 
-        StringBuilder message = new StringBuilder(1024);
-
-        // connection less packet
-        message.append('\u00ff');
-        message.append('\u00ff');
-        message.append('\u00ff');
-        message.append('\u00ff');
-
         // allow remote
+        // fixme: why?
         NET.Config(true);
 
-        message.append("rcon ");
+        // assemble password and arguments into a string and send
+        StringBuilder message = new StringBuilder(1024);
+        message.append(" ");
```

Comment:
does this change the protocol?

Reply:
good question! 
no it doesn't, as the `rcon` command type will be added in `Netchan.sendConnectionlessPacket(Defines.NS_CLIENT, to, ConnectionlessCommand.rcon, message.toString());`